pkg/cable/libreswan/libreswan.go

```diff
@@ -262,13 +262,15 @@ func (i *libreswan) ConnectToEndpoint(endpoint types.SubmarinerEndpoint) (string
 				args = append(args, "--id", localEndpointIdentifier)
 				args = append(args, "--host", localEndpointIP)
 				args = append(args, "--client", leftSubnets[lsi])
+				args = append(args, "--ikeport", i.ipSecNATTPort)
```

Comment:
Doesn’t this need to be handled differently for NAT and non-NAT?

```suggestion
if endpoint.Spec.NATEnabled {
    args = append(args, "--ikeport", i.ipSecNATTPort)
} else {
    args = append(args, "--ikeport", i.ipSecPort)
}
```

and same again for the right-hand side.

Reply:
Yes,  that makes sense.

Honestly I guess it doesn't really matter if we always go over a single port and remove the concept of nat/non-nat port ? 

I believe IKEv2 can do both over 4500 , may be all over "ipSecPort" and that's it?

Reply:
Yup, e2e works over 4500 in non-NAT mode.